scripts/peering-prefix

```diff
@@ -109,6 +138,15 @@ c)
     fi
     communities+=($OPTARG)
     ;;
+C)
+    for comm in $(echo $OPTARG | tr "," "\n")
+    do
+	if [[ $comm -gt 65535 || $comm -lt 1 ]] ; then
```

Comment:
Why do we only allow 16 bit numbers for these? Does BIRD not support BGP extended communities?

Reply:
I think BIRD supports extended communities, but given it is a bit of a mess in the wide-area Internet, I would be in favor of starting with 16-bit communities only.

Reply:
BIRD supports extended communities, but I chose to only implement the 16 bit version for the first version of the feature.